hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -223,6 +230,25 @@
         RACL error log information of this module.
       '''
     }
+    { struct:  "logic",
+      width:   "2"
+      type:    "uni",
+      name:    "sram_rerror",
+      act:     "req",
+      desc:    '''
+               SRAM read error log.
+               Bit 0: Correctable error
+               Bit 1: Uncorrectable error
```

Comment:
This encoding splits the information on whether there was _any_ error over two bits. I think it would be better to have one bit dedicated to the signaling of any error and a second bit to distinguish correctable from uncorrectable errors. As in:
Bit 0: SRAM integrity error detected during read
Bit 1: Error is correctable (0) or uncorrectable (1)